packages/libtirpc/project.bri

```diff
@@ -18,7 +18,7 @@ export default function libtirpc(): std.Recipe<std.Directory> {
     ./configure \\
       --prefix=/ \\
       --disable-gssapi
-    make -j16
+    make -j "$(nproc)"
```

Comment:
Just to notice here, I didn't set the env variable `OMP_NUM_THREADS` to override `nproc` command output. I think it's not necessary for this package.